src/main/java/org/jpeek/App.java

```diff
@@ -115,7 +115,9 @@ public void analyze() throws IOException {
         this.save(skeleton.toString(), "skeleton.xml");
         final Iterable<Report> reports = new ListOf<>(
             new Report(skeleton, "LCOM", this.params, 10.0d, -5.0d),
-            new Report(skeleton, "MMAC", this.params, 0.5d, 0.25d)
+            new Report(skeleton, "MMAC", this.params, 0.5d, 0.25d),
+            new Report(skeleton, "TCC",  this.params, 0.5d, 0.25d),
```

Comment:
@skapral do not declare them here if they are not finished and tested. And please, leave more puzzles: 1 for writing some tests for these reports and 1 for declaring them here once all the other puzzles are finished :)

Reply:
@amihaiemil fixed